src/main/java/org/broadinstitute/hellbender/tools/examples/ExampleVariantWalker.java

```diff
@@ -36,10 +37,17 @@
     public void onTraversalStart() {
         try {
             outputStream = outputFile != null ? new PrintStream(outputFile) : System.out;
-        }
-        catch ( final FileNotFoundException e ) {
+        } catch ( final FileNotFoundException e ) {
             throw new UserException.CouldNotReadInputFile(outputFile, e);
         }
+        if (readArguments.getReadInputs().stream().anyMatch(ReadInput::hasSymbolicName)) {
+            final Map<String, ReadInput> map = readArguments.getInputsBySymbolicName();
+            outputStream.println("reads inputs:");
+            for (final Map.Entry<String, ReadInput> kv : map.entrySet()){
+                outputStream.println("  " + kv.getKey() + ":" + kv.getValue().getReadFile().getName());
+            }
+            outputStream.println();
```

Comment:
It's strange that you chose to put this in `ExampleVariantWalker` -- recommend moving it to one of the `ExampleReadWalker*` classes